### PR TITLE
[#160648832] Card insertion layout

### PR DESCRIPTION
**Pull request by matgentili**
Changed locales strings

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#160648832](https://www.pivotaltracker.com/story/show/160648832): Come CIT, quando l'operazione di inserimento carta è terminata, non voglio proprio vedere la webview, voglio un semplice loader in overlay sulla app

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by matgentili** [comment]
Card insertion before:

card-insertion-before

Card insertion after:

card-insertion-after

**Comment by matgentili** [comment]
@Undermaken Very probably you're right, but if you want you can give me a card that requires a security code (I don't have it)  and I try to see if it works.

File: locales/en/index.yml [hunk 1]
```diff
@@ -303,7 +303,7 @@ wallet:
     addButton: Add a new method
     addDescription: You can add your payment methods (e.g. credit cards, bank accounts)
       so that you don't have to re-add them from scratch for every transaction
-    successful: The card was added successfully!
+    successful: Thanks, the new payment method has been entered
```
[the remark this answers is not in the document]
Comment (matgentili) [reply locales/en/index.yml]: Sorry I missed the comment

**Pull request closed without merging**